package.json

```diff
@@ -8,14 +8,15 @@
     "start": "next start",
     "lint": "next lint",
     "prepare": "husky",
-    "prettier": "prettier --write ."
+    "fmt": "prettier --write ."
```

Comment:
Any issues with this change? It felt easier to type out 😄

Reply:
👍